source/cloud-security/amazon/services/prerequisites/considerations.rst

```diff
@@ -125,12 +133,11 @@ The following example of a ``~/.aws/config`` file sets the supported configurati
    dev.proxy.use_forwarding_for_https = true
 
    dev.signature_version = s3v4
-   max_attempts = 5
-   retry_mode = standard
 
 .. note::
-   To configure multiple profiles for the integration, declare each profile in the ``~/.aws/config`` file using the same pattern as before.
-   If no profile is declared in the module configuration, the *default* profile is used.
+   The path to the ``s3`` and ``proxy`` sections, as well as ``signature_version``, must start with the profile name.
+
+To configure multiple profiles for the integration, declare each profile in the ``~/.aws/config`` file using the same pattern as before. If no profile is declared in the module configuration, the *default* profile is used.
```

Comment:
Instead of `is used`. I suggest `will be used` . 

```suggestion
To configure multiple profiles for the integration, declare each profile in the ``~/.aws/config`` file using the same pattern as before. If no profile is declared in the module configuration, the *default* profile will be used.
```